src/TestSuite/TestCase.php

```diff
@@ -189,6 +190,35 @@ public function loadFixtures()
         }
     }
 
+    /**
+     * simulate running the Application
+     * @param string|array $plugins list of Plugins to load
```

Comment:
Aren't we trying to avoid mixed mode arguments going forth?

Reply:
I will change it